service/service.py

```diff
@@ -108,6 +106,41 @@ def internal_server_error(error):
                    error='Internal Server Error',
                    message=message), status.HTTP_500_INTERNAL_SERVER_ERROR
 
+######################################################################
```

Comment:
can we use the earlier defined Flask-RESTPlus `api` objects? It was tailored to the assignment's requirements.

Reply:
I do not get what are you referring to in this comment.

Reply:
Our code contained Flask-RESTPlus instantiated before: https://github.com/nyu-devops-fall19-wishlists/wishlists/pull/182/files#diff-8bf0810d7919804091b3399999dc4b5fL38

Probably got lost due to merge conflicts.

Reply:
Yup. Changed back to what it was.